app/defaults.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Provide default values for the user session. These are automatically added
+ * via the `autoStoreData` middleware. These values will only be added to the
+ * session, if a value doesn't already exist.
+ * @type {Object}
```

Comment:
Could this comment include an example for how to use to set some data?

Reply:
Added this comment